serving/src/main/java/ai/djl/serving/ModelServer.java

```diff
@@ -459,6 +459,9 @@ private void initModelStore() throws IOException, BadWorkflowException {
                                     });
             if (configManager.waitModelLoading()) {
                 f.join();
+                if (stopped.get()) {
```

Comment:
We actually don't need this.

We can do:
```
            CompletableFuture<Void> f = modelManager.registerWorkflow(workflow);
            f.exceptionally(...)
```

Reply:
Done.